src/main/java/pl/wedding/management/model/employee/EmployeeDto.java

```diff
@@ -0,0 +1,18 @@
+package pl.wedding.management.model.employee;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@Builder
+@Getter
+class EmployeeDto {
+    private long id;
+    private String name;
+    private String surname;
+    private String workPlace;
+    private String position;
+    private BigDecimal salary;
```

Comment:
Zdaje się, że będziemy potrzebowali więcej informacji o wynagrodzeniu. "Salary" nie będzie wystarczające. Będziemy potrzebowali rodzaju zatrudnienia, sposobu rozliczenia (czy to za godzinę, czy opłata stała), dopiero wtedy kwotę. Opakuj te wszystkie informacje w nową klasę